connector/connect/common/src/main/protobuf/spark/connect/base.proto

```diff
@@ -778,6 +778,67 @@ message ReleaseExecuteResponse {
   optional string operation_id = 2;
 }
 
+message FetchErrorDetailsRequest {
+
+  // (Required)
+  // The session_id specifies a Spark session for a user identified by user_context.user_id.
+  // The id should be a UUID string of the format `00112233-4455-6677-8899-aabbccddeeff`.
+  string session_id = 1;
+
+  // User context
+  UserContext user_context = 2;
+
+  // (Required)
+  // The id of the error.
+  string error_id = 3;
+
+  // Specifies whether to include the stacktrace in the error message when
```

Comment:
what does this mean? Given the below structured data, why do you need to repeat the stack trace in the message itself?

Reply:
This is for Python Client

Reply:
The stack trace is not complicated algorithmic logic. Rendering this on the client should not be an issue. Having duplication of stacktrace_in_message does not make sense.

Reply:
https://github.com/apache/spark/pull/42377#discussion_r1330178093